docs/wxm-devices/D1-bundle/D1-contents.mdx

```diff
@@ -0,0 +1,108 @@
+---
+id: D1-contents
+title:  WeatherXM D1 bundle (WB1200) Box Contents
+sidebar_label: Contents
+slug: /wxm-devices/D1-bundle/D1-contents
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+## Included
+
+<div class="explanation">
+<img src={useBaseUrl("/img/wxm-devices/D1-bundle/contents-included-D1v2.svg")} 
+     style={{width: '50%', height: 'auto'}} />
+  <div>
+    <h4>Included in the Box</h4>
+    <ol>
+      <li>Wind speed sensor</li>
+      <li>Weather station</li>
+      <li>Wind vane</li>
+      <li>Metal plate</li>
+      <li>U-bolts (x2)</li>
+      <li>Nuts (x4)</li>
+      <li>Wrench</li>
+      <li>Screwdriver</li>
+      <li>WG1200 Gateway</li>
+      <li>Power supply cable (USB-C)</li>
+    </ol>
+  </div>
+</div>
+
+
+
+
+## Not Included
+
+You would also need:
+1. Batteries 1.5V **not rechargeable** (x2 AA)
+1. Metal mounting pole (recommended 50mm/2in pole diameter & at least 2m/6.56ft height)
+1. Tools and parts to secure the pole
+1. USB power supply
+
+:::danger
+If you use rechargeable batteries, you may harm your device
+:::
+
+<hr/>
+
+## Main Parts
+
+### Outdoor Weather Station
+
+The outdoor sensor is a set of sensors encased in a plastic shell that ensures the proper function of the weather station.
+
+The outdoor sensor is responsible for:
+
+1. the collection of measurements of local temperature, humidity, rain, wind speed and wind direction, and
+1. the wireless transmission of the data to the weather miner.
+
+<div class="explanation two-columns">
+  <img src={useBaseUrl("img/wxm-devices/wifi-m5-bundle/ws1000a_b-parts.svg")} />
+  <div>
+    <h4>Components</h4>
+    <ol>
+      <li>Wind Speed Sensor</li>
+      <li>Wind Vane</li>
+      <li>Thermo-hygro sensor</li>
+      <li>Rain collector</li>
+      <li>Bubble level</li>
+      <li>Solar panel</li>
+      <li>Antenna</li>
+      <li>Light sensor</li>
+      <li>U-Bolts</li>
+      <li>Battery compartment</li>
+      <li>Reset button</li>
+      <li>LED Indicator: light on for 4s on powers up. Then the LED will flash once every 16 seconds (the sensor transmission update period).</li>
+    </ol>
+  </div>
+</div>
+
+This part of your weather station is placed outdoors.
+For further details about an advisable installation of the outdoor sensor, please see [Installing the WS1000 Weather Station](/wxm-devices/wifi-m5-bundle/install-weather-station).
+
+The following figures show a sketch of the outdoor sensor. The sketch comes with a description of all the important parts and sensors of it.
+
+
+### WG1200 Gateway (D1 WiFi)
+
+<div class="control">
+  <img src={useBaseUrl("/img/home/wg1200-gateway.png")} />
```

Comment:
The image is pixel-ed, check this:
![εικόνα](https://github.com/WeatherXM/docs/assets/19330987/d9149dff-63c3-4458-a332-f2686fefb4d2)

Get a high res icon as it is too small and enlarging it makes it losing its quality.